### DISCUSSION TITLE
I can't do it cause of an error

### DISCUSSION THREAD

**Discussion by Ghostmanever** [Discord-OTP-Forcer/Discord-OTP-Forcer#143] [Community Support]
Sir, It's not working it's says ModuleNotFoundError: No module named 'webdriver_manager'. How can I fix this. I really need my 6 yrs old account. The discord Support team they just give up and said u have to delete the Account like what.
![Screenshot (28)](https://github.com/user-attachments/assets/03897881-0499-4023-a5b3-8fb1f139b6c5)

**Comment by Inbydev** [comment]
@Ghostmanever could you try with the latest commit?

https://github.com/Derpitron/Discord-OTP-Forcer/archive/refs/heads/main.zip

And follow the tutorial:

https://github.com/Derpitron/Discord-OTP-Forcer/wiki/How-to-setup-and-use
